components/patient-data/api/src/main/java/org/phenotips/data/Gene.java

```diff
@@ -60,7 +60,9 @@
     /**
      * The status of this gene for the described patient.
      *
-     * @return one of {@code solved}, {@code candidate}, {@code carrier}, {@code rejected_candidate} or {@code rejected}
+     * @return one of {@code solved}, {@code candidate}, {@code carrier}, {@code rejected_candidate}, {@code rejected},
+     *         {@code candidate_novel_disease}, {@code candidate_novel_phen}, {@code umc}, {@code umc_vus} or
+     *         {@code umc_msv}
```

Comment:
Comments should not list the available statuses, because they may change. Instead, a link to where the statuses are defined should be provided.